CHANGELOG.md

```diff
@@ -3,6 +3,74 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.23.3
+
+_Dec 19, 2024_
+
+We'd like to offer a big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Improve Korean locale on the Data Grid
+- 🐞 Bugfixes
+
+Special thanks go out to the community contributors who have helped make this release possible:
+@k-rajat19, @good-jinu.
+Following are all team members who have contributed to this release:
+@KenanYusuf, @MBilalShafi, @arminmeh, @flaviendelangle.
+
+<!--/ HIGHLIGHT_ABOVE_SEPARATOR /-->
+
+### Data Grid
+
+#### `@mui/x-data-grid@7.23.3`
+
+- [DataGrid] Allow to pass custom props to `.main` element (#15919) @MBilalShafi
```

Comment:
```suggestion
- [DataGrid] Allow passing custom props to `.main` element (#15919) @MBilalShafi
```

Reply:
Is this enough to make this a minor release?

Reply:
Based on changes, it looks more like a "bugfix". 🤔

Reply:
What do you think @MBilalShafi